CHANGELOG.rst

```diff
@@ -3,22 +3,51 @@ Changelog
 
 This document describes changes between each past release.
 
-10.2.0 (unreleased)
+11.0.0 (2018-10-09)
 -------------------
 
 **Breaking changes**
 
-- The ``basicauth`` policy is not used by default anymore
+- The ``basicauth`` policy is not used by default anymore (#1736)
+
+If your application relies on this specific behaviour, you now have to add explicitly settings:
+
+.. code-block:: ini
+
+    multiauth.policies = basicauth
+    multiauth.policy.basicauth.use = kinto.core.authentication.BasicAuthAuthenticationPolicy
```

Comment:
Is this necessary? We still have https://github.com/Kinto/kinto/blob/48d58f0fefe5c89218fb756616426c1efe06e05c/kinto/core/initialization.py#L111-L112